AndroidSDK/src/com/leanplum/LeanplumNotificationHelper.java

```diff
@@ -49,7 +50,7 @@
    */
   // NotificationCompat.Builder(Context context) constructor was deprecated in API level 26.
   @SuppressWarnings("deprecation")
-  static NotificationCompat.Builder getDefaultNotificationBuilder(Context context,
+  static NotificationCompat.Builder getDefaultCompatNotificationBuilder(Context context,
```

Comment:
NotificationCompat is available API22 and up only? are we handling this correctly then?

Reply:
NotificationCompat was added with 22 support library...

Reply:
Do we have to bump `compile "com.android.support:support-v4:[14.0.0,${SUPPORT_LIBRARY_VERSION}]"` to `compile "com.android.support:support-v4:[22.0.0,${SUPPORT_LIBRARY_VERSION}]"`